src/routes/_store/store.js

```diff
@@ -14,6 +14,7 @@ const persistedState = {
   currentRegisteredInstance: undefined,
   // we disable scrollbars by default on iOS
   disableCustomScrollbars: process.browser && /iP(?:hone|ad|od)/.test(navigator.userAgent),
+  bottomNav: false,
```

Comment:
Considering turning this on for all phone users (iOS and Android), but seems really tricky to detect properly. Folks can just turn this on themselves if they like.